pkg/requests/base.go

```diff
@@ -39,6 +40,21 @@ func (r *RequestParameters) SetStripeAccount(value string) {
 	r.stripeAccount = value
 }
 
+// RequestError captures the response of the request that resulted in an error
+type RequestError struct {
+	msg        string
+	StatusCode int
+	Body       map[string]interface{}
+}
+
+func (e RequestError) Error() string {
+	out, err := json.MarshalIndent(e.Body, "", "  ")
+	if err != nil {
+		panic(err)
```

Comment:
for safety maybe we should instead still return a string saying something like "unexpected error trying to marshal error's json body" or something.

Reply:
Sounds good. In that case, is the goal to print an error message for this particular request but still allow the event to continue executing?

Reply:
I think we should avoid `panic` in general and gracefully handle errors.

In that particular example, we're trying to stringify an error, so if json.MarshalIndent fails I would just return a simpler string, such as 
```
fmt.Printf("Request failed, %v", err)
```